### DISCUSSION TITLE
Edges to be connected even if the user drops it on the box and not only circle

### DISCUSSION THREAD

**Discussion by Sachin-chaurasiya** [xyflow/xyflow#2102] [Q&A] [answered]
We allow edge connection only if we drop edge on node circle, this takes many efforts because we have to do connection on that small circle.
Is it possible to make the connection when the user drops the edge on the box?
Thanks.

**Comment by bcakmakoglu** [comment, marked as answer]
There's multiple ways you could achieve that.
I guess the simplest would be to just have an invisible handle around your custom node that can catch the edges being dropped into the nodes.
[Here](https://codesandbox.io/s/sandpack-project-forked-mfbrjl?file=/ColorSelectorNode.js:240-247) is an example.
Try dropping edges into the custom node in the middle.

You could also just check `onConnectEnd` if it was triggered above a custom node and then manually add another edge, but that would be a little more complicated.
